src/docs/product/alerts-notifications/issue-alerts.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Issue Alerts
+sidebar_order: 2
+---
+
+Issue alerts apply only to error events, and trigger whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users. Sentry users with admin permissions or higher can create alerts. To confirm or set admin permissions, navigate to **Project Settings > Alerts > New Alert**. To change the frequency of your alerts, configure your [Alert Settings](/product/alerts-notifications/alert-settings/).
+
+## Conditions
+
+Conditions are evaluated for an issue alert **each time** the issue receives a new event, subject to [rate limits](/product/alerts-notifications/alerts/#rate-limit). Alert conditions have three parts:
+
+- Triggers ("When...")
+- Filters ("If...")
+- Actions ("Then...").
+
+Triggers specify what type of activity you'd like monitored. Filters help control noise by filtering down the triggered alerts to only those matching specific criteria. Actions specify what should happen when the alert passes the filters.
+
+![Alert condition builder](alert-conditions2.png)
+
+Note: Check for conflicting conditions. In the example below, the alert will not fire because an event **cannot satisfy both** of these conditions simultaneously: Sentry detects/creates a new issue, _and_ the issue has happened ten times.
+
+![The alert condition in this example shows the conditions are conflicting with each other.](alert-condition-conflict.png)
+
+Types of alert conditions:
+
+| Type | Condition |
```

Comment:
@adhiraj I'm thinking about breaking up this table into three separate tables for Trigger, Filter, and Action. The tables will each cover the dropdown options for each of the three parts. I think a table will be easier to digest than a bullet pointed list. Happy to discuss more.